src/index.js

```diff
@@ -0,0 +1,57 @@
+// eslint-disable-next-line max-classes-per-file
```

Comment:
- [ ] It would be better if we fix the ESLint errors instead of disabling it inline